products/health-care/beneficiary-travel/engineering/data-flow-sequence-diagram.md

```diff
@@ -0,0 +1,33 @@
+The basic sequence for most BTSSS is straightforward:
+
+```mermaid
+sequenceDiagram
+autonumber
+actor veteran as Veteran
+box VA.gov
+participant vets-website
+participant vets-api
+end
+participant travel-pay-api as Travel Pay API
+participant dynamics-db as Dynamics DB
+
+veteran->>vets-website: Navigate to travel pay
+
+activate vets-website
+
+vets-website->>vets-api: vets-api endpoints (/claims, /appointments, etc.)
+vets-api->>travel-pay-api: travel-pay endpoints (/claims, /appointments, etc.)
+travel-pay-api->dynamics-db: fetch data via dynamics portal plugins
```

Comment:
Keep in mind that these diagrams are from the perspective of our work on VA.gov. Dynamics is (or probably should be) hidden to us. I don't want to miss any nuance though - what's the benefit of us including the Dynamics system as part of this seq diagram?

Reply:
I guess it might not be necessary to include that final bit with Dynamics. My thinking has more to do with what I called out in the [non-diagram portion below](https://github.com/department-of-veterans-affairs/va.gov-team/pull/73133/files/e6401b311d57c70e08f437d2c61b5c71be46a97e#diff-91a82379c52f487631d0c9cfc93ebc184a168052b5634a156a4815d517a770f9). It's mainly to highlight that VA.gov and the existing portal are drawing from different DBs that don't talk to each other.

Reply:
I removed explicit mention of dynamics